modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -111,17 +118,24 @@ export interface CustomSigningFunction {
   (params: { coin: IBaseCoin; txPrebuild: TransactionPrebuild; pubs?: string[] }): Promise<SignedTransaction>;
 }
 
-export interface WalletSignTransactionOptions {
+export interface WalletSignBaseOptions {
```

Comment:
Name change is a BREAKING CHANGE.

Reply:
The name has not been changed. This has to do with the way the change shows up. Line 130 keeps the `WalletSignTransactionOptions` name